src/vivarium_gates_nutrition_optimization_child/data/loader.py

```diff
@@ -597,7 +658,15 @@ def load_underweight_exposure(key: str, location: str) -> pd.DataFrame:
         + ["stunting_parameter", "wasting_parameter", "parameter"]
     )
 
-    return df.sort_index()
+    # add wasting cat2.5 data by duplicating wasting cat2 data
+    cat2_rows = df.query("wasting_parameter=='cat2'").copy()
+    new_cat_rows = (
+        cat2_rows.reset_index()
+        .replace({"wasting_parameter": {"cat2": "cat2.5"}})
+        .set_index(df.index.names)
+    )
+    df = pd.concat([df, new_cat_rows]).sort_index()
+    return df
 
 
 def load_gbd_2021_exposure(key: str, location: str) -> pd.DataFrame:
```

Comment:
Remind me what we use `data_keys.WASTING.EXPOSURE` for. We use birth exposure to initialize the simulants and transitions to move them through the dynamic model. I believe we used exposure to manage the static portion of the wasting model, but haven't we introduced a dynamic model starting from birth now?

Reply:
We use the 1-5 month age group exposure data when defining birth prevalence. 

And yes, we should have a dynamic model from birth but I realize that is actually not how model 10 was implemented which is almost definitely the source of the bug.